tools/swift-format/Sources/Rules/NeverForceUnwrap.swift

```diff
@@ -8,5 +8,35 @@ import SwiftSyntax
 ///
 /// - SeeAlso: https://google.github.io/swift#force-unwrapping-and-force-casts
 public final class NeverForceUnwrap: SyntaxLintRule {
+  
+  // Checks if "XCTest" is an import statement
+  public override func visit(_ node: ImportDeclSyntax) {
+    var iterator = node.path.makeIterator()
+    while let component = iterator.next() {
+      if component.name.text == "XCTest" {
+        context.importsXCTest = true
```

Comment:
Ditto from NeverUseImplicitlyUnwrappedOptionals. Would like to see this cached on Context and see the visitor deduplicated.